### PR TITLE
Declaration/content changes

### PR DESCRIPTION
**Pull request by neil-stephen-mcgonigle**

### CODE REVIEW

File: src/lib/resource/sections/mainapplicant-declaration-under-12.js [hunk 1]
```diff
@@ -15,24 +15,23 @@ module.exports = {
...
...e</h2> <p class="govuk-body">It is not necessary to appoint a legal or other representative to act on a victim’s behalf. If a representative is appointed at any stage, please be aware that:</p> <ul class="govuk-list govuk-list--bullet"> <li>CICA cannot meet their costs </li> <li>we will only communicate directly with any appointed representative</li> </ul> <p class="govuk-body">If we make an award, we will pay it only to the victim or their legal representative. This is unless the application has been made on behalf of:</p> <ul class="govuk-list govuk-list--bullet"> <li>an adult who cannot manage their own financial affairs</li> <li>a child who is under 18 years of age</li> </ul> <p class="govuk-body"> It is our general policy to put an award for a child in an interest-earning deposit account until they reach the age of 18.</p> <p class="govuk-body"> If a monetary award is to be made and there is a dispute about outstanding legal fees, it is our policy to retain the disputed amount until the dispute has been resolved.</p> <p class="govuk-body"> If it is decided that a representative’s services are no longer required, CICA must be notified in writing as soon as possible.</p> </div>'
```
Comment (Julieallan) [inline src/lib/resource/sections/mainapplicant-declaration-under-12.js:18]: @neil-stephen-mcgonigle should these be p-mainapplicant-enter-your-name etc?

Comment (Julieallan) [inline src/lib/resource/sections/mainapplicant-declaration-under-12.js:18]: Last nested bullet, change 'you' to 'the victim'
Last bullet at top, change 'your' to 'the 'victim's'

**Review by Julieallan — CHANGES_REQUESTED** [review]

**Review by Julieallan — APPROVED** [review]

File: src/lib/resource/sections/mainapplicant-declaration-12-and-over.js [hunk 2]
```diff
@@ -15,24 +15,23 @@ module.exports = {
...
...e</h2> <p class="govuk-body">It is not necessary to appoint a legal or other representative to act on a victim’s behalf. If a representative is appointed at any stage, please be aware that:</p> <ul class="govuk-list govuk-list--bullet"> <li>CICA cannot meet their costs </li> <li>we will only communicate directly with any appointed representative</li> </ul> <p class="govuk-body">If we make an award, we will pay it only to the victim or their legal representative. This is unless the application has been made on behalf of:</p> <ul class="govuk-list govuk-list--bullet"> <li>an adult who cannot manage their own financial affairs</li> <li>a child who is under 18 years of age</li> </ul> <p class="govuk-body"> It is our general policy to put an award for a child in an interest-earning deposit account until they reach the age of 18.</p> <p class="govuk-body"> If a monetary award is to be made and there is a dispute about outstanding legal fees, it is our policy to retain the disputed amount until the dispute has been resolved.</p> <p class="govuk-body"> If it is decided that a representative’s services are no longer required, CICA must be notified in writing as soon as possible.</p> </div>'
```
Comment (Julieallan) [inline src/lib/resource/sections/mainapplicant-declaration-12-and-over.js:18]: @neil-stephen-mcgonigle should this be You have told us you are p-mainapplicant-enter-your-name etc?
Last nested bullet, change 'you' to he victim'
Last bullet at top, change 'your' to 'the victim's'

**Review by Julieallan — CHANGES_REQUESTED** [review]

File: src/lib/resource/sections/mainapplicant-declaration-under-12.js (hunk 1 above)
Comment (neil-stephen-mcgonigle) [reply src/lib/resource/sections/mainapplicant-declaration-under-12.js]: yes, they should!

**Review by Julieallan — APPROVED** [review]

**Review by Julieallan — APPROVED** [review]

**Pull request closed without merging**